mne/viz/tests/test_ica.py

```diff
@@ -408,6 +408,13 @@ def test_plot_ica_overlay():
     pytest.raises(TypeError, ica.plot_overlay, raw[:2, :3][0])
     pytest.raises(TypeError, ica.plot_overlay, raw, exclude=2)
     ica.plot_overlay(raw)
+
+    # Check that the issue addressed by the following pull request is fixed:
+    # https://github.com/mne-tools/mne-python/pull/11808
+    # Issue meant that an error would be raised by plot_sources, unless picks
+    # were consecutive, starting at 0.
+    ica.plot_sources(eog_epochs.average(), picks=1)
```

Comment:
oops, I got confused and guided you to the wrong place!  This is the `plot_overlay` test, we should add your new lines to the `plot_sources` test:

https://github.com/mne-tools/mne-python/blob/1704513f46f2596f7dccbb520e30804324999409/mne/viz/tests/test_ica.py#L266

Also a shorter comment would be ok, something like

```suggestion
    # regression test for `IndexError` when passing non-consecutive picks or consecutive
    # picks not including `0` (https://github.com/mne-tools/mne-python/pull/11808)
    ica.plot_sources(eog_epochs.average(), picks=1)
```

Reply:
Ahh, sorry I didn't notice that. I moved the test to the correct place. Thanks for editing the comment.